specifyweb/frontend/js_src/lib/components/FormSliders/IntegratedRecordSelector.tsx

```diff
@@ -115,6 +115,7 @@ export function IntegratedRecordSelector({
       <RecordSelectorFromCollection
         collection={collection}
         defaultIndex={isToOne ? 0 : index}
+        formType={formType}
         isCollapsed={isCollapsed}
```

Comment:
another small issue not caused by this PR:
you pass `isCollapsed` to RecordSelectorFromCollection, but that component does not need it (though it's in the TypeScript type)
can you please change the type to not need isCollapsed and see how that affects other places that are using the same type and might need isCollapsed

Reply:
Do you mean remove the type isCollapsed from type RecordSelectorProps?